### PR TITLE
KEYCLOAK-5299 Document how to explicitly set permitted hostnames

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

File: server_admin/topics/threat/host.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+=== Host
+
+{project_name} uses the request URL for a number of things. For example the URL sent in password reset emails.
+
+By default the request URL is based on the `Host` header and there is no check to make sure this URL is the valid and
+correct URL.
+
```
Comment (matthewhelmke) [inline server_admin/topics/threat/host.adoc]: You need a couple commas. In line 4, after "For example," and in line 6 after "By default". Also in line 9 after "invalid host headers."

File: server_admin/topics/threat/host.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </server>
+</subsystem>
+----
+
+The changes that have been made from the default config is to add the attribute `default-host="ignore"` and update the
+attribute `alias`. `default-host="ignore"` prevents unknown hosts from being handled, while `alias` is used to list the
+accepted hosts.
+
+Equivalent configuration using CLI commands:
+
```
Comment (matthewhelmke) [inline server_admin/topics/threat/host.adoc]: Here is the equivalent configuration using CLI commands:

**Review by matthewhelmke — COMMENTED** [review]
Looks good to me. I noted a few needed commas and made one suggestion.

**Comment by stianst** [comment]
All suggestions incorporated

**Pull request merged**